src/Converters.ts

```diff
@@ -9,7 +9,7 @@ export function fromRpcHttp(rpcHttp: rpc.RpcHttp$Properties) {
     query: rpcHttp.query,
     params: rpcHttp.params,
     body: fromTypedData(rpcHttp.body),
-    rawBody: rpcHttp.rawBody,
+    rawBody: fromTypedData(rpcHttp.rawBody, false),
```

Comment:
as mentioned in the other PR, might want to make this `string` only and avoid the boolean method flag